patches/tModLoader/Terraria/ModLoader/ILoadable.cs

```diff
@@ -11,6 +11,12 @@ public interface ILoadable
 		/// <param name="mod">The mod instance associated with this type.</param>
 		public abstract void Load(Mod mod);
 
+		/// <summary>
+		/// Whether or not this type should be loaded when it's told to. Returning false disables <see cref="Mod.AddContent"/> from actually loading this type
+		/// </summary>
+		/// <param name="mod">The mod instance trying to add this content</param>
+		public abstract bool IsLoadingEnabled(Mod mod);
```

Comment:
default interface implementation for this?

Reply:
Ah, forgot to change that back when I was messing with it, trying to get it to show up on ModTypes (I'm not too familiar with interfaces implementating each other). Fixed now